### PR TITLE
https://dreamy-fermat-e3204a.netlify.com/

### PR DESCRIPTION
**Pull request by Linamag92**

### CODE REVIEW

**Comment by Linamag92** [comment]
# Business Site

tried to practice making the menu, adapted for both computer and mobile on 2 different looks that I had some problems with from last week. then i added a picture and a sign up, where i have as you can see problems with my h2 and get started button not visible on the home page unfortunately. it showed up for a while, but managed to delete something so it disappeared. would get help from my mentor but he got sick unfortunately.


## The problem

If I had more time then I had made a new sign up form, and then I need to learn how to make it look clean among the codes, so feel free to give tips / links on how I learn this best.

and learn terminal and github better so i can upload/refresh my documents instead of write comments..

## View it live

https://dreamy-fermat-e3204a.netlify.com/

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,127 @@
...
 <html>
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8" /> <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+ <link href="/your-path-to-fontawesome/css/all.css" rel="stylesheet">
+ <link rel="stylesheet" href="style.css">
+ <link rel="stylesheet" href="script.js">
```
Comment (ericaglimsholt) [inline code/index.html]: The script tag is great to link in the bottom of the body tag actually!

File: code/index.html [hunk 2]
```diff
@@ -1,30 +1,127 @@
...
+              <input type="password"required autocomplete="off" placeholder="Set A Password"/>
+            </div>
+            <button type="submit" class="button button-block"/>Get Started</button>  
+</form>
+</div>   
+</form>
+</div>
+</div>
+</div>
+</div>
```
Comment (ericaglimsholt) [inline code/index.html]: It would be great if these closing tags is indented, then you will easier see where it opens! 😀

File: code/script.js [hunk 3]
```diff
@@ -1,9 +1,75 @@
...
+
+
+
 function toggle() {
   this.classList.toggle("active")
 }
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+$('.form').find('input, textarea').on('keyup blur focus', function (e) {
```
Comment (ericaglimsholt) [inline code/script.js]: Seems like that you are using a framework called jQuery here! That's great if you want, but you can do the same with vanilla javascript as well if you want to practice it!

**Pull request closed without merging**